eip155/caip25.md

```diff
@@ -113,23 +129,20 @@ Similarly, wherever possible, session properties should align closely with infor
       }
     },      
     "sessionProperties": {
-      "capabilities": {
-          "0xab16a96d359ec26a11e2c2b3d8f8b8942d5bfcdb": {
-            "0x31": {
-              "paymasterService": {
-                 "supported": true
-               },
-               "sessionKeys": {
-                  "supported": true
-               }
-            },
-            "0x3432313631": {
-              "sessionKeys": {
-                "supported": false
-              }              
-            }
-          }
-       }   
+      "expiry": "2022-12-24T17:07:31+00:00",
+      "caip154": {
+        "supported":"true"
+      },
+      "flow-control": {
+        "supported": "true"
+      }
```

Comment:
Would it be more beneficial to define the flow-control capabilities already in here instead of just sending a flag?

E.g.:
- flow-control in sessionProperties defined globally for all sessionScopes:
```
"sessionProperties": {
	"flow-control": {
		 "loose": ["halt", "continue"],
          	"strict": ["continue"]
	}
}
```

- flow-control in scopedProperties defined per chain:
```
"scopedProperties": {
	"84532"{
		"flow-control": {
		 	"loose": ["halt", "continue"],
          		"strict": ["continue"]
		}
	}
}
```

It would reduce round trips between the peers

Does it make sense to you all?

Reply:
Oh, I was just following exactly what's in the example in today's 5792; i thought the exact options for flow-control (and even dash versus camelcase) were still being debated OOB!

Reply:
Agree on the example, looking good. This way peers can already exchange the flow-control capabilities aka we're reducing a need to call `wallet_getCapabilities`

Reply:
update [in this commit](https://github.com/ChainAgnostic/namespaces/pull/139/commits/37dc500ac61d2dd4d87fcc07f05b58d7a7cc471d) to use the selective example above